packages/edit-post/src/components/layout/index.native.js

```diff
@@ -115,7 +115,7 @@ class Layout extends Component {
 		// Add a margin view at the bottom for the header.
 		const marginBottom =
 			Platform.OS === 'android' && ! isHtmlView
-				? headerToolbarStyles.container.height
+				? headerToolbarStyles[ 'header-toolbar__container' ].height
```

Comment:
This value takes the default height of the header toolbar, so it doesn't take into account the case when the toolbar is displayed in the expanded mode. As far as I tested, I didn't identify any visual issue regarding this logic, so looks safe to only use that value.

In the future, it would be great to update this calculation, and instead of accessing the height from the styles, retrieve it from the `onLayout` event.